packages/angular/src/generators/storybook-configuration/schema.json

```diff
@@ -42,9 +42,10 @@
       "enum": ["eslint", "none"],
       "default": "eslint"
     },
-    "projectBuildConfig": {
-      "description": "Provide a custom projectBuildConfig for the Angular executor. If left blank, Nx will use the default.",
-      "type": "string"
+    "tsConfiguration": {
+      "type": "boolean",
+      "description": "Configure your project with TypeScript. Generate main.ts and preview.ts files, instead of main.js and preview.js.",
+      "default": false
     }
```

Comment:
Should we add an alias to this for "projectBuildConfig" if there is no migration? This would break people using the existing flag name right?

Reply:
No, no, it's something totally different! I just did two birds with one stone. Removed the old flag which is not used anywhere, and added this new one, which is irrelevant!

Reply:
Thanks for the review, btw!! :)

Reply:
Yes yes, your comment was super helpful, I just created a [PR](https://github.com/nrwl/nx/pull/10664) to remove that flag from everywhere, to avoid further confusion!! :D :D